### DISCUSSION TITLE
Test result not found

### DISCUSSION THREAD

**Discussion by geekox86** [vitest-dev/vscode#68] [Q&A]
Greetings and thank you for the wonderful library.
I can run the tests and they pass from the terminal.
However, when I start any test form the test explorer or from the editor, I get an error message saying "Test result not found".
Also, when debugging a test I get an error in the debug console saying:
```
/usr/local/bin/node ./node_modules/vitest/vitest.mjs /Users/geekox86/Documents/Projects/vitest-test/a.test.ts -t some test --api 55154
 DEV  v0.20.2 /Users/geekox86/Documents/Projects/vitest-test
      API started at http://localhost:55154
Could not read source map for file:///Users/geekox86/Documents/Projects/vitest-test/@vite/env: ENOENT: no such file or directory, open '/Users/geekox86/Documents/Projects/vitest-test/@vite/env.mjs.map'
 ✓ a.test.ts  (1 test) 2155ms
Test Files  1 passed (1)
     Tests  1 passed (1)
      Time  4.94s (setup 0ms, collect 358ms, tests 2.15s)

 PASS  Waiting for file changes...
       press h to show help, press q to quit
```
The test indicator keeps spinning until I stop the debug session by hand.

BTW, I also cloned your repo to test with your samples and the issue still exists.

macOS Big Sur 11.6.5
Version: 1.70.0-insider

"typescript": "^4.7.4",
"vite": "^3.0.4",
"vitest": "^0.20.2"

Please, help.

**Comment by pyncz** [comment]
It looks like you specified "test" command in your package.json as `vitest` instead of `vitest run`, so it runs in watch mode (as default) and waits for changes, so you get infinite loading
